src/kagglehub/models.py

```diff
@@ -22,6 +22,7 @@ def model_download(handle: str, path: Optional[str] = None, *, force_download: O
         A string representing the path to the requested model files.
     """
     h = parse_model_handle(handle)
+    logger.info(f"Downloading Model: {handle}")
```

Comment:
This is quite spammy especially with libraries like Keras which download files one-by-one. We had a similar log statement initially but removed it.

Reply:
Is the spam specifically a problem for the stdout/stderr? We can filter the messages that the console stream doesn't output anything, but the RotatingFile will contain all entries.

Reply:
Yes, specifically for stdout/stderr. Also, I would suggest testing inside Kaggle Notebook and Colab because the logger is configured differently with this environment to avoid reintroducing the log spam issue we fixed.

Reply:
You can use this code block:

```
classifier = keras_nlp.models.BertClassifier.from_preset(
    "bert_base_en",
    num_classes=4,
)
```

And look at the output share the output in a `go/paste` for each environment.

Reply:
See b/337291048#comment3